prov/psm/src/psmx.h

```diff
@@ -96,8 +96,10 @@ extern int psmx_am_compat_mode;
 #define PSMX_DOM_CAPS	(FI_LOCAL_COMM | FI_REMOTE_COMM)
 
 #define PSMX_MAX_MSG_SIZE	((0x1ULL << 32) - 1)
+#define PSMX_RMA_ORDER_SIZE	(2032)	/* Same as PSMX_AM_CHUNK_SIZE */
```

Comment:
Wouldn't it be better to define this as PSMX_AM_CHUNK_SIZE directly then?

Reply:
Was trying to keep related definitions close to each other. But I can make the change.